src/pages/shopping-add-image.svg

```diff
@@ -0,0 +1 @@
-156 -79.47583)" fill="#2f2e41"/><circle cx="353.41268" cy="284.48709" r="23.91104" fill="#ffb8b8"/><path d="M488.06354,513.40689,550.403,511.699l-18.30772-57.61963c11.37738-18.23924-.073-36.46-16.27789-54.67684l-12.80948-8.53965-20.49517-1.70793h0a50.3141,50.3141,0,0,0-13.54915,35.0504C469.36939,452.03191,474.29652,482.324,488.06354,513.40689Z" transform="translate(-156 -79.47583)" fill="#575a89"/><path d="M631.29146,552.19152h0a10.59327,10.59327,0,0,0-2.74926-16.96611L602.4085,522.15112l-9.567,11.441,24.36342,19.395A10.59327,10.59327,0,0,0,631.29146,552.19152Z" transform="translate(-156 -79.47583)" fill="#ffb8b8"/><path d="M599.933,545.85758l10.24759-24.765-55.50168-34.99377-23.37023-72.268a16.33817,16.33817,0,0,0-18.43188-11.054h0a16.33815,16.33815,0,0,0-12.42089,21.79283l17.92341,62.79126Z" transform="translate(-156 -79.47583)" fill="#575a89"/><path d="M470.13027,354.5693c0,9.30389,6.04222,23.7124,13.66345,29.8888,5.12781,4.15568,11.90457,3.98066,14.51741-1.70793,2.68032-5.83557,11.91945-19.55229,28.18086-28.18087,21.67761-11.50252-7.68756-31.18152-28.18086-28.18086C482.9115,328.64329,470.13027,339.00544,470.13027,354.5693Z" transform="translate(-156 -79.47583)" fill="#2f2e41"/><circle cx="328.64768" cy="245.20468" r="17.07931" fill="#2f2e41"/><path d="M504.28889,318.70275a17.06865,17.06865,0,0,0-15.79836-17.01463c.42437-.03158.84854-.06468,1.28094-.06468a17.07931,17.07931,0,0,1,0,34.15862c-.4324,0-.85657-.0331-1.28094-.06468A17.06869,17.06869,0,0,0,504.28889,318.70275Z" transform="translate(-156 -79.47583)" fill="#2f2e41"/><ellipse cx="353.83966" cy="287.90296" rx="3.84284" ry="5.12379" fill="#ffb8b8"/><polygon points="837.751 684.845 571.879 684.845 515.578 418.694 467.529 418.694 467.529 406.752 525.258 406.752 581.559 672.902 837.751 672.902 837.751 684.845" fill="#3f3d56"/><circle cx="610.84102" cy="707.87676" r="25.59138" fill="#3f3d56"/><circle cx="801.92331" cy="707.87676" r="25.59138" fill="#3f3d56"/><path d="M988.90348,758.69673H730.22506L684.58373,533.14489h347.15138a12.2655,12.2655,0,0,1,12.11592,14.16689l-42.8333,201.0251A12.20374,12.20374,0,0,1,988.90348,758.69673Zm-255.86937-3.41219H988.90348a8.80568,8.80568,0,0,0,8.74205-7.47748l42.8333-201.0251a8.85014,8.85014,0,0,0-8.74372-10.22489H688.72567Z" transform="translate(-156 -79.47583)" fill="#3f3d56"/><polygon points="624.513 678.499 594.623 455.653 597.989 455.097 627.879 677.943 624.513 678.499" fill="#3f3d56"/><polygon points="790.712 676.788 787.343 676.242 816.36 455.102 819.729 455.648 790.712 676.788" fill="#3f3d56"/><rect x="708.11491" y="455.37515" width="3.41218" height="222.84575" fill="#3f3d56"/><rect x="865.45079" y="431.52231" width="3.41219" height="330.29569" transform="translate(112.38887 1382.90398) rotate(-89.86127)" fill="#3f3d56"/><rect x="718.83766" y="700.96181" width="290.1779" height="3.41226" transform="translate(-160.7831 -73.5538) rotate(-0.39166)" fill="#3f3d56"/><ellipse cx="472.64757" cy="411.01676" rx="18.76701" ry="10.23655" fill="#3f3d56"/></svg>
```

Comment:
Beautiful Illustration! 💚 

What do you think about creating a folder: `assets` and subfolder like: `illustrations`, `icons` (like SVG files)? I believe it could help with your project organization. Only an idea. 😄

Reply:
That's a great idea! We can do that next week in a small improvement issue. That way we don't cause too many conflicts during the merge.